vtex-apps/store-components
# Feature: SKU Selector `displayPrices` prop

#### What problem is this solving?

-  Adds the option to show the SKU variation price in SKU Selector. This is best suited when `displayMode` is set to `select` but it works as well in `default` mode as shown bellow;

#### How to test it?

1. Open [Workspace](https://juliomoreira--drinkrunnr.myvtex.com/);
1. Scroll until the section "Summer";
1. Interact with the SKU Selector;
1. Observe that the item price is showing;

#### Screenshots or example usage:

##### Product Summary

![ezgif com-gif-maker (1)](https://user-images.githubusercontent.com/1207017/100929775-e5e39b80-34c6-11eb-91a8-66867c5b1a3f.gif)

##### Product Page

![Screen Shot 2020-12-02 at 17 31 13](https://user-images.githubusercontent.com/1207017/100929937-22af9280-34c7-11eb-9011-b2ed0475d213.png)

#### How does this PR make you feel? [:link:](http://giphy.com/)



![](https://media.giphy.com/media/7WyUkDrDlOFf1Q8ODi/giphy.gif)
